squarelet/email_api/viewsets.py

```diff
@@ -0,0 +1,122 @@
+# Django
+from django.shortcuts import get_object_or_404
+
+# Third Party
+from allauth.account import signals
+from allauth.account.forms import AddEmailForm
+from allauth.account.models import EmailAddress, EmailConfirmation, EmailConfirmationHMAC
+from rest_framework import mixins, status, viewsets
+from rest_framework.response import Response
+
+# Squarelet
+from squarelet.email_api.serializers import PressPassEmailAddressSerializer
+
+
+class PressPassEmailConfirmationViewSet(
```

Comment:
For a one off endpoint like this, it makes more sense to use an `APIView` (https://www.django-rest-framework.org/api-guide/views/) then a `ViewSet` which is designed for handling multiple CRUD endpoints at once